kura/org.eclipse.kura.linux.net/src/main/java/org/eclipse/kura/linux/net/util/LinuxNetworkUtil.java

```diff
@@ -1124,7 +1152,7 @@ public void createApNetworkInterface(String ifaceName, String dedicatedApInterfa
             return;
         }
 
-        CommandStatus status = this.executeCommand(formIwDevIfaceInterfaceAddAp(ifaceName, dedicatedApInterface));
+        CommandStatus status = executeCommand(formIwDevIfaceInterfaceAddAp(ifaceName, dedicatedApInterface));
```

Comment:
Why did you remove `this`? Shouldn't our Kura cleanup profile enforce the use of `this` for member variables? Does it change behaviour in case of static classes?

Reply:
AFAIK, the clenup enforces the `this.` prefix only for variables, not for methods. Maybe the cleanup fix it?

Reply:
> Maybe the cleanup fix it?

That's probably it 👍